frontend/app/services/GoogleDirectoryService.scala

```diff
@@ -0,0 +1,66 @@
+package services
+
+import java.io.{File, FileInputStream}
+
+import com.amazonaws.regions.{Regions, Region}
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
+import com.google.api.client.http.javanet.NetHttpTransport
+import com.google.api.client.json.jackson2.JacksonFactory
+import com.google.api.client.util.SecurityUtils
+import com.google.api.services.admin.directory.model.Group
+import com.google.api.services.admin.directory.{Directory, DirectoryScopes}
+
+import configuration.Config
+import scala.collection.JavaConverters._
+import scala.concurrent._
+
+case class GoogleDirectoryConfig(
+    serviceAccountId: String,
+    serviceAccountEmail: String,
+    serviceAccountCert: String)
+
+object GoogleDirectoryService {
+
+  val transport = new NetHttpTransport()
+  val jsonFactory = new JacksonFactory()
+
+  val credential = new GoogleCredential.Builder()
+    .setTransport(transport)
+    .setJsonFactory(jsonFactory)
```

Comment:
You might be able to get rid of these two (setTransport & setJsonFactory), as we're using service flow, not using a refreshing token?